src/Components/Tasks/Tasks.tsx

```diff
@@ -187,231 +220,523 @@ const Tasks: React.FC = () => {
-                  )}
-                </div>
-                <div className="form-group my-3">
-                  <label className="label-title mb-2">Description</label>
-                  <textarea
-                    {...register("description", {
-                      required: true,
-                    })}
-                    rows={5}
-                    type="text"
-                    name="description"
-                    className="form-control"
-                    placeholder="Enter description..."
-                  ></textarea>
-
-                  {errors.title && errors.title.type === "required" && (
-                    <span className="text-danger ">desciption is required</span>
-                  )}
-                </div>
-                <div className="form-group my-3">
-                  <select
-                    {...register("employeeId", { required: true, valueAsNumber: true })}
-                    aria-label="Default select example"
+                  </thead>
+                  <tbody>
+                    {tasks.map((task: any) => (
+                      <tr key={task?.id}>
+                        <th scope="row">{task?.title}</th>
 
-                    className="form-select"
-                  >
-                    <option className="text-muted">
-                      User
-                    </option>
-                    {userList.map((user) => (
-                      <option key={user.id} value={user.id} >
-                        <td>{user.userName}</td>
+                        <td className=' text-white' style={{ textAlign: 'center' }}>
+                          <div style={{
+                            backgroundColor:
+                              task?.status === 'ToDo'
+                                ? '#E4E1F5'
+                                : task?.status === 'InProgress'
+                                  ? '#EF9B28'
+                                  : task?.status === 'Done'
+                                    ? '#009247'
+                                    : 'inherit',
+                            borderRadius: '15px',
+                            fontSize: '16px',
+                            padding: '10px', // Adjust the padding as needed
+                            fontWeight: '250',
+                            fontFamily: 'Montserrat-Regular',
+                          }}>
+                            {task?.status}
+                          </div>
+                        </td>
+                        <td>{task?.description}</td>
+                        <td>{task.employee.userName}</td>
+                        <td>{task.project.title}</td>
+                        <td>{new Date(task.creationDate).toLocaleDateString()}</td>
+                        <td>
+                          <button className="p-0 border-0 bg-white">
+                            <i
+                              onClick={() => showViewModal(task?.id)}
```

Comment:
The onClick should be on the button not the icon to enable users perform the action using the spacebar or the Enter key.